FusionIIIT/applications/academic_information/views.py

```diff
@@ -216,6 +216,11 @@ def homepage(request):
         return HttpResponseRedirect('/academic-procedures/')
 
     context = get_context(request)
+    semestor_nos=[]
```

Comment:
Instead of sending the semester array via context, can you simply iterate over 1 to 9 in frontend itself